internal/commands/app/diff.go

```diff
@@ -73,7 +73,7 @@ func (cmd *CommandDiff) Flags() []flags.Flag {
 			Value: &cmd.inputs.IncludeHosting,
 			Meta: flags.Meta{
 				Name:      "include-hosting",
-				Shorthand: "h",
```

Comment:
this (previous) change was the instigator for this PR... i very casually flipped the shorthand from `-s` to `-h` in [this commit](65101cd5674d741fb0dbc53bf71d97c2beac9e9c) without thinking too much.  of course, `-h` is already reserved as the shorthand for help text, so these commands panic'd when run.

i'm reverting these back (I shouldn't have changed them without expecting to do a major release in the first place 😬 ), and the test lays down some preliminary groundwork for what an "e2e test" might look like for these commands.

the `guidelines.md` is a result from the project retro, where it was mentioned documenting a lot of these command patterns would be beneficial for developers down the line

Reply:
[honest q] do we actually need a shorthand for this (and maybe some other commands)? `-s` isn't really a super intuitive shorthand imo (unless I'm missing something), would it be terrible if we just forced users to provide the full `--include-hosting` command... or just `--hosting`, for example?

Reply:
im with you, and my answer would be "no" (and likely just avoid shorthands all together until they're explicitly called/requested for).

i just felt like preserving the initially released behavior with `2.0.0` was more preferable than making this a "breaking change" (though i doubt anyone actually uses it this way, currently)

Reply:
cool I feel ya. yeah, agreed, let's not force a 3.0 version bump just because we changed a shorthand flag 😆